tests/anonymous-credentials/presentation.spec.ts

```diff
@@ -504,6 +521,58 @@ describe(`${Scheme} Presentation creation and verification`, () => {
     expect(presJson).toEqual(recreatedPres.toJSON());
   });
 
+  it('from credential1 - presenting pseudonyms', () => {
+    const builder1 = new PresentationBuilder();
```

Comment:
The encoding is the problem. Add this code here as a test
```
const decodedBaseForSecretKey = PseudonymBases.decodeBaseForSecretKey(base1ForSecretKey);
    const encodedBaseForSecretKey = PseudonymBases.encodeBaseForSecretKey(decodedBaseForSecretKey);
expect(base1ForSecretKey).toEqual(encodedBaseForSecretKey);
```

Reply:
Done